src/app/pipes/img-src/img-src.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ApiConfigurationService } from 'src/app/services/api-configuration.service';
+
+@Pipe({
+  name: 'imgSrc',
+})
+export class ImgSrcPipe implements PipeTransform {
+  constructor(private apiConfigService: ApiConfigurationService) {}
+
+  private apiConfig = this.apiConfigService.getConfig();
+  private defaultImgSize = 'w185';
+
+  /* todo: find a solution how to accept different size[] from apiConfig
+  depending on what image type passed into pipe */
```

Comment:
Just use Enum of possible values. Let's discuss this offline

Reply:
Updated, but don't like my solution, please suggest improvements.